lib/kernel/src/rpc.erl

```diff
@@ -40,7 +40,7 @@ some specific side effects on the remote node.
 > implementation also provides a more scalable implementation with better
 > performance than the original `rpc` implementation. However, since the
 > introduction of `erpc`, the `rpc` module implements large parts of its central
-> functionality using `erpc`, so the `rpc` module won't not suffer scalability
+> functionality using `erpc`, so the `rpc` module won't suffer scalability
 > wise and performance wise compared to `erpc`.
```

Comment:
I think this whole sentence should be rephrased, it sounds rather awkward. Especially the term "suffer" sounds wrong to me. IMO it would mean that the `rpc` module could potentially _become worse_ in terms of scalability and/or performance _because of_ the introduction of `erpc` (unless it did implement its functionality using `erpc`).

On a side note, I believe the correct use of "wise" in this context (meaning "in terms of") would be as a suffix, "scalabilitywise" and "performancewise" (which isn't too easy on the eye, either). When used as a single word or tacked on with a hyphen, the meaning of "wise" is "knowledgeable about" (as in "wisdom"). I may be wrong here, though, I'm not a linguist :sweat_smile: